src/testsuite/vec_f128_dummy.c

```diff
@@ -269,42 +269,44 @@ _test_xfer_bin128_2_vui16t (__binary128 f128)
 /* Mostly compiler and library tests follow to see what the various
  * compilers will do. */
 
-#ifndef __clang__ // to disable __clang__ because <math.h> fails
+#ifndef PVECLIB_DISABLE_F128MATH
+// Enable only if math.h supports generic C11 functions for __float128
+// __clang__ has a bug whenever -mfloat128 is enabled, maybe clang 10.0.1
```

Comment:
delete this comment?

Reply:
Keep this too because of the note about possible fix in clang 10.1